pwiz_tools/Skyline/Controls/Graphs/SummaryRelativeAbundanceGraphPane.cs

```diff
@@ -158,7 +158,13 @@ private void UpdateFormatting(IEnumerable<MatchRgbHexColor> colorRows)
         public void OnLabelOverlapPropertyChange(object sender, PropertyChangedEventArgs e)
         {
             if (e.PropertyName == @"GroupComparisonAvoidLabelOverlap")
```

Comment:
This should probably have been:
```
if (e.PropertyName == nameof(Settings.Default.GroupComparisonAvoidLabelOverlap))
```
so that "Find References" and other sorts of refactoring would work a little bit better.